theseus/geometry/se3.py

```diff
@@ -623,6 +625,165 @@ def to(self, *args, **kwargs):
         super().to(*args, **kwargs)
         self._resolve_eps()
 
+    def _deprecated_log_map_impl(
```

Comment:
I thought we are only making a deprecated version of the 'else' block and making the 'if' block the main version. Why is the whole thing deprecated?

Reply:
+1. At the very least we don't need the whole `if/else` block here, only the branch that is no longer available in the new code. But I think it's better if we encapsulate just the part that we are removing. Something like:

```python
def _log_map_impl(
        self, jacobians: Optional[List[torch.Tensor]] = None
    ) -> torch.Tensor:
    ...
    stuff = self.compute_things_for_log_map(some_inputs)
    ...
    # use stuff to compute more stuff


def _deprecated_compute_things_for_log_map(self, some_inputs):
    # the old way to do this
```

Reply:
Refactored.